### ISSUE TITLE
Configuration file has malformed JSON

### ISSUE THREAD

**Issue by msandfor** [all-contributors/all-contributors#445]
**Describe the bug**

"This project's configuration file has malformed JSON: .all-contributorsrc. Error:: Unexpected token ] in JSON at position 2843"

**To Reproduce**
Error occurs after standard request to add a user as a contributor. Several previously successfully added.

**Additional context**
After this occured the first time, I uninstalled the bot and deleted the .all-contributorsrc and removed the code from the readme. And then started again. But the problem reoccurred - not on the same user

Is there already a solution/known bug for this? I apologise if there is, but I could not find it.

**Comment by Berkmann18** [comment]
Sounds like a malformed JSON file that you need to fix or pass through `all-contributors-cli` to get the auto fixer to fix what it can repair.

**Comment by msandfor** [comment]
Um, is a @Berkmann18 is it a stupid question to ask how to do that? It was just a one click button to install the bot on my repo. I don't see anything about the `all-contributors-cli' - is it in the docs somewhere?

**Comment by Berkmann18** [comment]
@msandfor Not really, have you looked at [documentation](https://allcontributors.org/docs/en/overview)?

**Comment by msandfor** [comment]
@Berkmann18 oh yeah, I find it - those are nicely written docs. I'll try and give it a go later. I've uninstalled the bot from my project for now, but it's a cool bit of automation so keen to get it working.

**Comment by Berkmann18** [comment]
@codess-aus Did you managed to resolve the issue?
